dss-pades/src/main/java/eu/europa/esig/dss/pdf/PdfMemoryUsageSetting.java

```diff
@@ -0,0 +1,205 @@
+/**
+ * DSS - Digital Signature Services
+ * Copyright (C) 2015 European Commission, provided under the CEF programme
+ * 
+ * This file is part of the "DSS - Digital Signature Services" project.
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package eu.europa.esig.dss.pdf;
+
+/**
+ * Represents the PDF document loading setting.
+ * NOTE: The setting is applicable only when an instance of {@code eu.europa.esig.dss.model.FileDocument}
```

Comment:
It's not true, the setting is applied also with other types of Document

Reply:
Yes, you are right. It is indeed applicable in other cases too (mainly for PdfBox implementation),